mvlearn/decomposition/groupica.py

```diff
@@ -0,0 +1,320 @@
+    n_components : int, optional
+        Number of components to extract. If None, n_components is set to
+        the minimum number of features in the dataset
+
+    n_individual_components : int or list of int or 'auto', optional
+        The number of individual components to extract as a preprocessing.
+        If None, no preprocessing is applied. If an `int`, each dataset
+        is reduced to this dimension. If a list, the dataset `i` is
+        reduced to the dimension `n_individual_components[i]`.
+        If `'auto'`, set to the minimum between n_components and the
+        smallest number of features in each dataset.
+
+    multiple_outputs : bool, optional
+        If True, the `.transform` method returns one dataset per view.
+        Otherwise, it returns one dataset, of shape (n_samples, n_components)
+
+    prewhiten : bool, optional (default False)
+        Whether the data should be whitened after the original preprocessing.
+
+    solver : str {'picard', 'fastica'}
+        Chooses which ICA solver to use. `picard` is generally faster and
+        more reliable.
+
+    ica_kwargs : dict
+        Optional keyword arguments for the ICA solver. If solver='fastica',
+        see the documentation of sklearn.decomposition.fastica.
+        If solver='picard', see the documentation of picard.picard.
+
+    random_state : int, RandomState instance, default=None
+        Controls the random number generator used in the estimator. Pass an int
+        for reproducible output across multiple function calls.
+
+    Attributes
+    ----------
+    means_ : list of arrays of shape (n_components,)
+        The mean of each dataset
+
+    grouppca_ : mvlearn.decomposition.GroupPCA instance
+        A GroupPCA class for preprocessing and dimension reduction
+
+    unmixing_ : array, shape (n_components, n_components)
+        The square unmixing matrix, linking the output of the group-pca
+        and the independent components.
+
+    components_ : array, shape (n_components, n_components)
+        The square mixing matrix
+
+    individual_components_ : list of array
+        Individual unmixing matrices estimated by least squares.
+        `individual_components_[i]` is an array of shape
+        (n_components, n_features) where n_features is the number of
+        features in the dataset `i`.
+
+    individual_mixing_ : list of array
+        Individual mixing matrices estimated by least squares.
+        `individual_components_[i]` is an array of shape
+        (n_features, n_components) where n_features is the number of
+        features in the dataset `i`.
+
+    n_components_ : int
+        The estimated number of components.
+
+    n_features_ : list of int
+        Number of features in each training dataset.
+
+    n_samples_ : int
+        Number of samples in the training data.
+
+    n_subjects_ : int
+        Number of subjects in the training data
+
+    See also
+    --------
+    grouppca
+    multiviewica
```

Comment:
can you fix the case so the link hopefully work?

Reply:
done